### ISSUE TITLE
Please document how to exactly build the WASM version of the  tree-sitter grammar and where to put that file 

### ISSUE THREAD

**Issue by pdenapo** [staysail/zed-d#5]
The README.md file says

"You need to build the WASM version of the Tree-Sitter grammar, and then put that in the grammars folder, and then copy this repo into the appropriate extension folder." 

This is not clear enough. Please document exactly how to do that. It would be nice to provide a shell script for doing so automatically.

**Comment by maxdeviant** [comment]
Those steps seem to be a little outdated.

None of that should be necessary now.

You can either:

1. Install the D extension from the extension registry (you do have to scroll a little ways, since lots of extensions have the letter "d" in them)
    - Alternatively, you can add the following to your `settings.json`:
      ```json
      "auto_install_extensions": {
        "d": true
      },
      ```
2. Use "Install Dev Extension" from within Zed and point it at this repo, and it will build it from source.

**Comment by gdamore** [comment]
Yeah, I should fix those instructions.  For almost *everyone* they should just 'install' the extension right from the editor.

**Comment by gdamore** [comment]
In fact, the README already says this:

> Just use the zed: install dev extension command on a directory containing a checkout of this repository.

I don't see any references to building WASM explicitly.  Did you have an older version checked out?

**Comment by pdenapo** [comment]
Seems to be fixed now. It was an old version of the README in Github.
